src/target/stm32f4.c

```diff
@@ -239,6 +246,39 @@ bool stm32f4_probe(target_s *t)
 	return false;
 }
 
+bool gd32f4_probe(target_s *t)
+{
+	if ((t->part_id != ID_GD32F450) && (t->part_id != ID_GD32F470))
```

Comment:
The inner braces are redundant, please remove them as in `if (t->part_id != ID_GD32F450 && t->part_id != ID_GD32F470)` as they suppress a GCC warning we want here